.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
         },
     },
     rules: {
+        'import/no-extraneous-dependencies': 'off',
```

Comment:
it throws an error that your using devdependecies, rather than normal ones. lots of our js is for dev, so seems silly to have to keep disabling it?